### DISCUSSION TITLE
What is the right syntax for data-gjs-stylable?

### DISCUSSION THREAD

**Discussion by ltenti-eventboost** [GrapesJS/grapesjs#3474] [Q&A] [answered]
Hi everyone,

I'm trying to understand which is the right syntax for data-gjs-stylable, I tried the following, but none is functioning:

data-gjs-stylable="'width','margin','margin-top','margin-left','margin-right','margin-bottom'"
data-gjs-stylable="['width','margin','margin-top','margin-left','margin-right','margin-bottom']"
data-gjs-stylable="width,margin,margin-top,margin-left,margin-right,margin-bottom"

**Comment by artf** [comment]
This should work `data-gjs-stylable="['width','margin','margin-top','margin-left','margin-right','margin-bottom']"` if it doesn't, try to create a reproducible demo

**Reply by ltenti-eventboost** [reply]
@artf you can find a demo here: https://codepen.io/Gigi72/pen/GRWvKex

**Reply by artf** [reply]
ah ok, this is related to the JSON parser
```js
JSON.parse(`["key1", "key2"]`) // ok
JSON.parse(`['key1', 'key2']`) // SyntaxError
```
so, instead of ` data-gjs-prop="['key1', 'key2']"` you should use ` data-gjs-prop='["key1", "key2"]'`

**Reply by ltenti-eventboost** [reply]
It makes sense and I confirm that's the solution.
Thank you @artf !

**Comment by ltenti-eventboost** [comment, marked as answer]
ah ok, this is related to the JSON parser

JSON.parse(`["key1", "key2"]`) // ok
JSON.parse(`['key1', 'key2']`) // SyntaxError
so, instead of data-gjs-prop="['key1', 'key2']" you should use data-gjs-prop='["key1", "key2"]'
